src/test/java/us/warframestat/moddetection/api/AppTest.java

```diff
@@ -27,8 +29,12 @@ void shouldAnswerWithTrue()
 
     @Test
     void testModDetection() throws IOException {
+        API.setupData();
         BufferedImage image = ImageIO.read(Objects.requireNonNull(this.getClass().getResourceAsStream("/images/1.jpg")));
-        Map.Entry<BufferedImage, Integer> output = DetectMods.run(image, "cascade19", 1.150, 4, "pc");
+        Map.Entry<BufferedImage, JSONObject> output = DetectMods.run(image, "cascade19", 1.150, 4, "pc");
+        LOGGER.info(output.getValue().toString());
+        LOGGER.info(String.valueOf(output.getValue().toString().length()));
```

Comment:
Is there any risks of these loggers failing due to NullPointerExceptions? Are these calls guaranteed to be an object at minimum?

Reply:
thankfully it's in a test, but yeah, should probably do some asserts that they're not null